src/components/MenuItems.tsx

```diff
@@ -240,6 +240,12 @@ const SettingsMenuListItem = () => {
         <ListItemText primary="Settings" />
       </ListItem>
       <Collapse in={isExpanded} timeout="auto" unmountOnExit>
+        <ListItem component={NavLink} to="/Units" button>
+          <ListItemIcon>
+            <ProposalSettingsIcon />
```

Comment:
Are we using the same icon for both Proposal statuses and Units?

Reply:
Fixed,